trino-connector/trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/iceberg/IcebergPropertyMeta.java

```diff
@@ -44,6 +44,15 @@ public class IcebergPropertyMeta implements HasPropertyMeta {
   /** Property key for table location configuration. */
   public static final String ICEBERG_LOCATION_PROPERTY = "location";
 
+  /** Property key for table format configuration. */
+  public static final String ICEBERG_FORMAT_PROPERTY = "format";
```

Comment:
Is it `iceberg.file-format `?  https://trino.io/docs/current/connector/iceberg.html#general-configuration